easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb

```diff
@@ -0,0 +1,137 @@
+name = 'Python'
+version = '2.7.11'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.10.4'
+scipyversion = '0.17.0'
```

Comment:
I will choose the ones you prefer. Just tell me which versions should be added here...